### ISSUE TITLE
Provide a default device independent mode

### ISSUE THREAD

**Issue by yschimke** [Splitties/ComposeOClock#7]
Maybe similar to WFF, some mode based on a theoretical watch, say 450px and 225dp, that ensures that watchfaces look consistent across devices.

Comparison 
![image](https://github.com/Splitties/ComposeOClock/assets/231923/7ec8d173-f372-4f72-bc81-53d6b00a5a89)
![image](https://github.com/Splitties/ComposeOClock/assets/231923/99855c9d-0ff0-4a39-8511-ef6c26ea60be)

**Comment by LouisCAD** [comment]
By default in `OClockRootCanvas`, size should currently be capped between 192 and 227 dp (sizes of small and large round Wear previews in Compose Wear tooling artifacts).

It's possible to pass a custom `Modifier` to have a specific size that either matches one of those two values, or some arbitrary watch (including one of the user is using), or something based on the phone/foldable screen width and/or some user setting.

I think the size you want to present depends on what you want the UI using it to look like.

Do you have a specific idea of doing something in a particular place within this sample project?

**Comment by yschimke** [comment]
@LouisCAD No, just that maybe the default should be this device independent mode.

So when you design carefully but only for a single device, you aren't surprised on other devices.

Then opt into responsive designs when you choose.

**Comment by LouisCAD** [comment]
It really depends on your choices designing the Watch Face.

If you use fractions of `size`, it'll always look the same, just bigger or smaller based on the actual device screen size.

If you use dp sizes, you'll get fixed size and more padding on large screens, and less on smaller ones.

That's up to the designer, and that example is showing both in the same design.
